src/components/dev-hub/podcast-player/audio-player.js

```diff
@@ -0,0 +1,205 @@
+            grid-area: total;
+            margin-right: ${size.tiny};
+            text-align: end;
+        }
+    }
+`;
+
+const ContentContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+`;
+
+const PlayerContainer = styled('div')`
+    align-items: flex-start;
+    background-color: ${colorMap.devBlack};
+    border-radius: ${size.xsmall};
+    border: ${colorMap.greyDarkTwo} 1px solid;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    padding: ${size.large};
+    width: 100%;
+`;
+
+const AudioContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    width: 100%;
+
+    @media ${screenSize.upToMedium} {
+        flex-direction: column;
+
+        ${ControlsContainer} {
+            a:last-of-type {
+                margin-right: 0;
+            }
+        }
+    }
+`;
+
+const StyledSliderContainer = styled('div')`
+    margin-left: ${size.default};
+    width: 100%;
+
+    @media ${screenSize.upToMedium} {
+        margin-top: ${size.small};
+        margin-left: 0;
+    }
+
+    @media ${screenSize.upToSmall} {
+        margin-top: ${size.mediumLarge};
+
+        ${SliderContainer} {
+            ${mobileSliderStyles}
+        }
+    }
+`;
+
+const PlayerFooter = styled('div')`
+    margin-top: ${size.small};
+
+    @media ${screenSize.upToMedium} {
+        margin-top: ${size.large};
+        width: 100%;
+    }
+`;
+
+const AudioPlayer = ({ className, podcast }) => {
+    const playerRef = useRef(null);
+    const [playerState, playerDispatch] = useReducer(
+        playerReducer,
+        playerInitialState
+    );
+    const { isPlaying, progress, duration, isReady } = playerState;
+
+    const setDuration = useCallback(duration => {
+        playerDispatch({ type: PLAYER_ACTIONS.SET_DURATION, value: duration });
+    }, []);
+
+    const setProgress = useCallback(progress => {
+        playerDispatch({ type: PLAYER_ACTIONS.SET_PROGRESS, value: progress });
+    }, []);
+
+    const toggleIsPlaying = useCallback(e => {
+        e.stopPropagation();
+        playerDispatch({
+            type: PLAYER_ACTIONS.TOGGLE_IS_PLAYING,
+        });
+    }, []);
+
+    const onChangeHandler = useCallback(
+        seconds => {
+            if (isReady) {
+                setProgress({ playedSeconds: seconds });
+                playerRef.current.seekTo(seconds);
+            }
+        },
+        [isReady, setProgress]
+    );
+
+    const onReadyHandler = useCallback(player => {
+        playerRef.current = player;
+        playerDispatch({
+            type: PLAYER_ACTIONS.SET_IS_READY,
+            value: true,
+        });
+    }, []);
+
+    const moveForwardHandler = useCallback(() => {
+        if (isReady) {
+            const { getCurrentTime, seekTo } = playerRef.current;
+            const newTime = getCurrentTime() + 30;
+            seekTo(newTime, 'seconds');
+        }
+    }, [isReady]);
+
+    const moveBackHandler = useCallback(() => {
```

Comment:
![Screen Shot 2021-07-30 at 2 47 02 PM](https://user-images.githubusercontent.com/9064401/127698373-4605cea7-8e54-442d-befd-887f7b03afa8.png)

I noticed the hover for the forward and rewind might also need to move the text as well if possible?

Reply:
what is text here ? 30 ?